actions/src/main/java/com/redhat/service/bridge/actions/webhook/WebhookInvoker.java

```diff
@@ -0,0 +1,46 @@
+package com.redhat.service.bridge.actions.webhook;
+
+import java.io.IOException;
+
+import org.slf4j.Logger;
+import org.slf4j.LoggerFactory;
+
+import com.redhat.service.bridge.actions.ActionInvoker;
+
+import okhttp3.MediaType;
+import okhttp3.OkHttpClient;
+import okhttp3.Request;
+import okhttp3.RequestBody;
+import okhttp3.Response;
+
+public class WebhookInvoker implements ActionInvoker {
```

Comment:
Any reason not to make this bean `@ApplicationScoped` and then injected into the `WebHookAction` rather than creating an instance per-invocation of the Action?

Reply:
I just replicated the same structure as the `KafkaTopicAction`...I think if we want to change this we should define a structure first and be consistent among all actions.

Reply:
Thanks for pointing out keeping things consistent. Lets go with it 👍